.github/workflows/terraform.yml

```diff
@@ -0,0 +1,40 @@
+name: 'Terraform'
+
+on:
+  push:
+    branches:
+    - main
+  pull_request:
+
+jobs:
+  terraform:
+    name: 'Terraform'
+    runs-on: ubuntu-latest
+    environment: production
+    env:
+      AWS_ACCESS_KEY_ID: ${{ secrets.ACCESS_KEY_ID }}
+      AWS_SECRET_ACCESS_KEY: ${{ secrets.SECRET_ACCESS_KEY }}
+
+    defaults:
+      run:
+        shell: bash
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: install Terraform
+      run: wget https://releases.hashicorp.com/terraform/0.15.5/terraform_0.15.5_linux_amd64.zip && unzip terraform_0.15.5_linux_amd64.zip && export PATH=$PATH:$(pwd)
+
+    - name: Terraform Init
+      run: terraform init
+
+    - name: Terraform Format
+      run: terraform fmt -check
+
+    - name: Terraform Plan
+      run: terraform plan
+
+    - name: Terraform Apply
+      if: github.ref == 'refs/heads/main' && github.event_name == 'push'
+      run: terraform apply -auto-approve
```

Comment:
I see that there's no human revision of the plan right on the PR in the worflow as it is. You can find this step in these references:

- https://learn.hashicorp.com/tutorials/terraform/github-actions?in=terraform/automation#review-actions-workflow
- https://github.com/marketplace/actions/hashicorp-setup-terraform

Basically, they take the output of the `terraform plan` command and shows it directly on the PR to be easier to review. Do you think it is worth adding it?

Reply:
This is great, I will add

Reply:
the github actions is done, i need to create a token on github. I will do this on monday.